utils/utils.go

```diff
@@ -57,3 +57,17 @@ func FuzzyAgo(ago time.Duration) string {
 func Spinner() *spinner.Spinner {
 	return spinner.New(spinner.CharSets[11], 400*time.Millisecond)
 }
+
+// ColorFuncForState returns a color function for a PR/Issue state
+func ColorFuncForState(state string) func(string) string {
```

Comment:
Since `colorFuncForState` was refered from both `command/issue.go` and `command/pr.go`, I moved it to `utils.go`.

Reply:
Nit: could you move this back to the `command` package, since it's only needed there? We're trying to slim down the `utils` package until we can remove it.

I can live in either `issue.go` or `pr.go`; it doesn't matter.